server/cloud_secrets.py

```diff
@@ -22,3 +22,10 @@ def get(secret):
     name = "projects/{0}/secrets/{1}/versions/latest".format(constants.PROJECT_ID, secret)
     payload = secret_client.access_secret_version(name=name).payload.data.decode("UTF-8")
     return payload
+
+
+def get_or_none(secret):
+    try:
+        return get(secret)
+    except:
```

Comment:
PEP8 frowns on bare exception clauses.

https://github.com/python/peps/blob/master/pep-0008.txt

See line 1180.

Can you add the exception that's raised on a missing secret here?

Reply:
Yep. I'll do that.